cloudsync/sync/manager.py

```diff
@@ -178,35 +178,44 @@ def set_root_oid(self, side, oid):
     def set_resolver(self, resolver):
         self._resolve_conflict = resolver
 
+    def sync_one_entry(self, sync: SyncEntry):
```

Comment:
How are we going to handle errors on the CryptVFS side when syncing down a file? Will we have to be listening to the notification manager at the same time as calling `smart_sync_ent`

Reply:
errors are synchronous now